src/test/functional/shared/apiHelpers.ts

```diff
@@ -18,12 +19,34 @@ export const createUserWithRoles = async (email, password, forename, userRoles)
       {
         headers: {'Content-Type': 'application/json'},
       })).data;
-
   } catch (e) {
     throw new Error(`Failed to create user ${email} with roles ${userRoles}, http-status: ${e.response?.status}`);
   }
 };
 
+export const createUserWithSsoId = async (email, password, forename, userRoles,ssoId) => {
```

Comment:
Missing white space between userRoles and ssoId

Reply:
Fixed